definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-choose-service-application-nonprod.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "LiveFrom": "27/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "awaitingServicePayment",
+    "UserRole": "caseworker-divorce-courtadmin-la",
```

Comment:
I think you might need to remove this from the normal AuthorisationCaseState.json. Otherwise there would be a duplicate, and add the one in the normal AuthorisationCaseState to AuthorisationCaseState-prod.json.

For existing permissions (i.e. if a permission was in the normal AuthorisationCaseState.json), you will need to move it to AuthorisationCaseState-prod.json if you want to add a different permission in the equivalent -nonprod.json file

Reply:
Yes, you must not have duplicates in "core" files and "prod" "nonprod" files.